.travis.yml

```diff
@@ -0,0 +1,4 @@
+language: node_js
+node_js:
+  - 6
```

Comment:
just fyi that's a very low version of nodejs these days
![image](https://user-images.githubusercontent.com/38820281/64229295-bb560c00-cf2c-11e9-9726-79c4e558be0e.png)
but it doesn't hurt supporting old versions if that's your aim

Reply:
Ha, okay. I can go to 8, it's just what the tests run as on travis anyway.